### PR TITLE
Fix symlinks

### LINKED ISSUE #11: Do not try to recreate symlinks if they already exist
First run:

```
Creating mirrors directories...
Creating logs directories for nginx...
Creating logs directories for cron...
```

Second run onwards:

```
ln: failed to create symbolic link '/var/www/web/mirrors/mirrors': File exists
ln: failed to create symbolic link '/logs/toran-proxy': File exists
```

### PR DESCRIPTION
**Pull request by teohhanhui**
Previous fix #12 was botched. This fixes #11

### CODE REVIEW

File: bin/install/toran.sh [hunk 1]
```diff
@@ -62,11 +63,14 @@ else
...
 
 # Create directory mirrors
-if [ ! -e $DATA_DIRECTORY/mirrors ]; then
+if [ ! -d $DATA_DIRECTORY/mirrors ]; then
     echo "Creating mirrors directories..."
     mkdir -p $DATA_DIRECTORY/mirrors
 fi
-ln -sfn $DATA_DIRECTORY/mirrors $WORK_DIRECTORY/web/mirrors
+if [ -d $WORK_DIRECTORY/web/mirrors ]; then
+    rm -rf $WORK_DIRECTORY/web/mirrors
```
Comment (cedvan) [inline bin/install/toran.sh:71]: It should not be losing data already existing

Comment (teohhanhui) [inline bin/install/toran.sh:71]: Now that I go through the codes (both `toran` and yours), I don't see what's the purpose of this. `/var/www/web/mirrors` is not used anywhere.

Comment (cedvan) [inline bin/install/toran.sh:71]: error sorry, the `rm` is ok. It is useful to Toran Proxy to download packages ;)

**Comment by cedvan** [comment]
It's ok 

Thanks you to contributing ;)

File: bin/install/toran.sh (hunk 1 above)
Comment (teohhanhui) [inline bin/install/toran.sh:71]:
``` yaml
# assets/config/config.yml
git_path: /data/toran-proxy/mirrors
```

is used provided the config is being read, which it is not thanks to a bug(?) in Toran (`toran_config_file` parameter is hardcoded at the top of the bundle's `services.yml`)

and `/var/www/web/repo` is also used by Toran.

But `/var/www/web/mirrors` is not being used.

Comment (cedvan) [inline bin/install/toran.sh:71]: I think that composer need presence of this folder (symbolic link) in the same path as the folder repo

**Pull request merged**